src/components/inputs/text/ValidationInput/ValidationInput.tsx

```diff
@@ -66,31 +78,90 @@ function ValidationInput({
     );
 
     setValidationResults(validationResults);
+    const validity = validationResults.every((value) => value.status === true);
+    setValid(validity);
     if (setIsValid) {
-      setIsValid(validationResults.every((value) => value.status === true));
+      setIsValid(validity);
```

Comment:
why do we have `setIsValid` and `setValid` - what's the difference?

Reply:
one is local state, another is a callback function